### PR TITLE
Rebased #227 - David Chadwick's fixes for disputed credentials

### PR DESCRIPTION
**Pull request by dlongley**
Replaces #227.

***
<a href="https://pr-preview.s3.amazonaws.com/w3c/vc-data-model/pull/259.html" title="Last updated on Nov 12, 2018, 5:48 PM GMT (ea58b6b)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/vc-data-model/259/4bcfb99...ea58b6b.html" title="Last updated on Nov 12, 2018, 5:48 PM GMT (ea58b6b)">Diff</a>

### CODE REVIEW

**Comment by David-Chadwick** [comment]
LGTM

File: index.html [hunk 1]
```diff
@@ -1929,23 +1929,52 @@ <h2>Disputes</h2>
...
 The time may come when an <a>entity</a> wants to dispute a <a>credential</a>
-issued by another entity. The mechanism for doing this is the same as issuing
-a regular credential except that the <a>subject</a> identifier for the
-<a>claims</a> are those of the disputed credential. For example, if a disputed
-credential with an identifier of
-<code>http://con-artist.example.com/credentials/3732</code>
-contains disputed statements, an entity would issue the following credential
-in a public venue to make it known that the credential is disputed:
-        </p>
+issued by the issuer. There at least two different cases to consider:
```
Comment (gannan08) [inline index.html]: Missing verb.
```suggestion
issued by the issuer. There are at least two different cases to consider:
```

Comment (dlongley) [reply index.html]: Fixed

**Comment by msporny** [comment]
Hold off merging until Nov 1st for more reviews.

**Comment by stonematt** [comment]
also #226 ?

**Pull request merged**